build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/info/GlobalBuildInfoPlugin.java

```diff
@@ -353,6 +369,26 @@ private Provider<File> findRuntimeJavaHome() {
         });
     }
 
+    private Provider<File> resolveEarlyAccessJavaHome(Integer runtimeJavaProperty) {
+        NamedDomainObjectContainer<Jdk> container = project.getPlugins().apply(JdkDownloadPlugin.class).getContainer(project);
```

Comment:
Applying the plugin like this _seems_ wrong but maybe my spidey sense is tingling here for no good reason.

Reply:
yeah feeling dirty. fixed it :D